### DISCUSSION TITLE
How to add &nbsp;

### DISCUSSION THREAD

**Discussion by markusgritsch** [lona-web-org/lona#362] [Q&A]
In a LonaView I would like to return two Buttons separated by some `&nbsp;` like this:
```Python
    html = HTML(
        Button('B1'),
        '&nbsp;',
        Button('B2'),
        )
```
However, this gets rendered as:
![grafik](https://user-images.githubusercontent.com/2684183/225710139-5413e223-292a-4433-aa24-3fa6a2e5951f.png)

Other HTML tags like `<br>` or `<b>` get interpreted as markup, and even the following works:
```Python
    html = HTML(
        Button('B1'),
        '<b>te&nbsp;xt</b>',
        Button('B2'),
        )
```
![grafik](https://user-images.githubusercontent.com/2684183/225710867-c7b4f06c-2389-450a-8324-da0fc61a5ceb.png)

I think this is related to issue https://github.com/lona-web-org/lona/issues/268, so it there still no way to achive this?

**Comment by fscherf** [comment]
Hi @markusgritsch!

This problem is known, and is no parsing problem, but a rendering problem. It is related to #343 which, not #268.

Coincidentally i am currently working on patch-set, with multiple changes for the client, that contains two patches, which should fix this problem: https://github.com/lona-web-org/lona/commit/799612cd8ad80504a288254d75eb8af8c030f11c (branch is named fscherf/topic/new-widget-api).

Could you test whether this fixes your issue?

**Comment by markusgritsch** [comment]
Indeed, with your new branch it works fine :)
I'm impressed again, and thanks for fixing this already in advance :D

**Reply by fscherf** [reply]
Thanks :) And thanks for testing :) I will request your review, when the PR is ready. I think the series should be ready sometime this weekend. I will do a release after it.

I will convert this discussion to an issue, and link the PR to it
